src/app/pages/viewing/responses-list/responses-list.component.ts

```diff
@@ -43,22 +47,39 @@ export class ResponsesListComponent implements OnInit {
     }
   }
 
-  ngOnInit(): void { }
+  ngOnInit(): void {
+    if (this.cookieService.check('viewing-cookie') === false) {
+      this.cookieService.set('viewing-cookie', 'true', 365);
+    }
+
+    this.showNewToViewingValue = this.cookieService.get('viewing-cookie');
+
+    if (this.showNewToViewingValue === 'true') {
+      this.showNewToViewing = true;
+    } else if (this.showNewToViewingValue === 'false') {
+      this.showNewToViewing = false;
+    } else {
+      console.log('Error. Viewing-cookie not found or initialized properly.');
```

Comment:
```suggestion
     AuthService.displayError('Viewing-cookie not found or initialized properly.');
```

Reply:
@janakitti  / @TommyHuang-dev you'll have to double check if this works, I edited this in from the web editor.